src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.*;
```

Comment:
Maybe you should import only ArrayList instead of everything from the java util library?